source/docs/networking/networking-introduction/roborio-network-troubleshooting.rst

```diff
@@ -1,61 +1,101 @@
 
 If this ping succeeds, but the .local ping fails, it is likely that either the roboRIO hostname is configured incorrectly, or you are connected to a DNS server which is attempting to resolve the .local address.
 
 - Verify that your roboRIO has been :ref:`imaged for your team number <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`. This sets the hostname used by mDNS.
-- Disconnect your computer from all other networks including Ethernet and WiFi. It is possible that one of these networks contains a DNS server that is attempting to resolve the .local address.
+- :ref:`Disable all other network adapters <docs/networking/networking-introduction/roborio-network-troubleshooting:Disabling Network Adapters>`
 
 Ethernet Connection
--------------------
+^^^^^^^^^^^^^^^^^^^
 
 .. image:: images/roborio-troubleshooting/roborio-ip-address.png
 
 If you are troubleshooting an Ethernet connection, it may be helpful to first make sure that you can connect to the roboRIO using the USB connection. Using the USB connection, open the :ref:`roboRIO webdashboard <docs/software/roborio-info/roborio-web-dashboard:roboRIO Web Dashboard>` and verify that the roboRIO has an IP address on the ethernet interface. If you are tethering to the roboRIO directly this should be a self-assigned ``169.*.*.*`` address, if you are connected to the OM5P-AN radio, it should be an address of the form ``10.TE.AM.XX`` where TEAM is your four digit FRC team number. If the only IP address here is the USB address, verify the physical roboRIO ethernet connection.
 
-Ping the roboRIO IP address
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If there is an IP address in the step above, try pinging this IP address using the command prompt as described above. If this works, you have an issue resolving the mDNS address on your PC. The two most common causes are not having an mDNS resolver installed on the system and a DNS server on the network that is trying to resolve the .local address using regular DNS.
-
-- Verify that you have an mDNS resolver installed on your system. On Windows, this is typically fulfilled by the NI FRC Game Tools. For more information on mDNS resolvers, see the :doc:`roboRIO Networking article <networking-basics>`.
-- Disconnect your computer from any other networks and make sure you have the OM5P-AN configured as an access point, using the :ref:`FRC Radio Configuration Utility <docs/zero-to-robot/step-3/radio-programming:Programming your Radio>`. Removing any other routers from the system will help verify that there is not a DNS server causing the issue.
+Disabling Network Adapters
```

Comment:
This should remain a sub-section underneath ethernet troubleshooting.

Reply:
I disagree that the ethernet connection subsection is a required parent of this.  It is mostly about looking at the IP address of the roboRIO on the WebUI and that is a much more complicated step and more unlikely to be wrong (everything assigns the RIO to be .2).  The two things to learn from the Ethernet connection step IMO are did someone set an incorrect static address OR is the address link local (which means you are connected via USB and the ethernet cable isn't plugged in or working).  Both of these are more complex and less essential IMO of the intial identification of if the users mDNS is working or not.

Reply:
I'll agree with oyu on that.